docs/api-reference/layers/editable-geojson-layer.md

```diff
@@ -354,3 +354,29 @@ The pointer moved, regardless of whether the pointer is down, up, and whether or
 * `groundCoords` (Array): `[lng, lat]` ground coordinates where the pointer is now.
 * `isDragging` (Boolean): `true` if the pointer is moving but it is considered a drag, in this case you likely want to handle `onDragging`
 * `pointerDownPicks` (Array): An array containing [deck.gl Picking Info Objects](https://uber.github.io/deck.gl/#/documentation/developer-guide/adding-interactivity?section=what-can-be-picked-) for all objects that were under the pointer when it went down, or `null` if pointer is moving without pointer down.
+
+
+## Define Sublayers
+
+Three default sublayers make up the composite `EditableGeoJsonLayer`:
+
+1. `GeometryLayer` renders the geometries. Default: `GeoJsonLayer`
+2. `PointHandleLayer` renders the point edit handles. Default: `ScatterplotLayer`
+3. `IconHandleLayer` renders the icon edit handles. Default `IconLayer`
+
+All three sublayer defaults can be overridden with custom or experimental layers. Define an object prop that contains the `Layer` class object, an `id` and any addtional `props`. For instance:
+
+```
+PointHandleLayer: {
+  Layer: OutlinedScatterplotLayer,
+  id: 'outlined',
+  props: {
+    innerRadius: 5,
+    fillColor: [0x20, 0x40, 0x90, 0xc0],
+    strokeColor: [0x0, 0x0, 0x0, 0x80]
```

Comment:
I think previously we discussed that we want more advanced ways to pass attributes.
Like, how can I define the `fillColor` based on the current feature?

@supersonicclay What do you think?

Reply:
Agreed, we need some way to provide the same callbacks. For example, someone should be able to do `StrokedScatterplotLayer`  and specify: `getColor: (f, isSelected) => isSelected ? RED : GRAY`. I don't see a way to accomplish that here.

Reply:
Okay, I'll look into this.

Reply:
So I was thinking each prop is either a value or a function of the form:

`getProperty([feature[, isSelected[, currMode]]])`

Then every prop which is a function is made aware with something like:

`getProperty: feature => this.props.getProperty(feature, isSelected, this.props.mode)`

Basically the same logic as your `selectionAwareAccessor` wrapper, but also `mode` aware. The only thing I'm unsure of is how to flag the props which are callbacks. Maybe they get their own definition object, separate from `props`?

Reply:
Maybe something like this?

```js
new EditableGeoJsonLayer({
  data,
  editHandlesLayer: {
    layer: IconLayer,
    props: {
      iconAtlas: 'myIcons.png',
      getSize: (editHandle, context) => {
        if (editHandle.type === 'existing') {
          return 20;
        } else {
          return 10;
        }
        console.log(context.isSelected);
        console.log(context.mode);
    },
    accessors: [
      'getSize'
      // any accessors here will have `context` argument added to the end of the arguments
    ]
  }
  
})
```